src/ui/preferences/GeneralSettings.qml

```diff
@@ -140,7 +140,7 @@ QGCView {
                         //-----------------------------------------------------------------
                         //-- Base UI Font Point Size
                         Row {
-                            visible: _appFontPointSize.visible
+                            visible: _appFontPointSize ? _appFontPointSize.visible : false
```

Comment:
It was throwing a Javascript exception because _appFontPointSize wasn't defined.